### ISSUE TITLE
Are we actually choosing the best model from independent starts?

### ISSUE THREAD

**Issue by jgallowa07** [matsengrp/torchdms#173]
I see that we run some number of independent starts like [this](https://github.com/matsengrp/torchdms/blob/70e0a5076bc54e3689300b8b3ead90f5c3b6de98/torchdms/analysis.py#L289)

But it seems to me this just chooses the last of the independent starts, not the _best_ ... But maybe I'm missing something in our clever analysis class. When looking at the output of some models using independent starts, I can confirm that the best validation loss is not the model being chosen for full training ...

**Comment by matsen** [comment]
I am about to run to a meeting, but the way this worked once was that a model only got saved to the output file if it was the best thus far. Then that model got loaded in for the final training.
